src/Twig/Extension/SonataAdminExtension.php

```diff
@@ -143,14 +144,46 @@ public function renderListElement(
             $environment
         );
 
-        return $this->output($fieldDescription, $template, array_merge($params, [
+        return $this->render($fieldDescription, $template, array_merge($params, [
             'admin' => $fieldDescription->getAdmin(),
             'object' => $object,
             'value' => $this->getValueFromFieldDescription($object, $fieldDescription),
             'field_description' => $fieldDescription,
         ]), $environment);
     }
 
+    /**
+     * return the value related to FieldDescription, if the associated object does no
+     * exists => a temporary one is created.
+     *
+     * @param object $object
+     *
+     * @throws \RuntimeException
+     *
+     * @return mixed
+     */
+    public function getValueFromFieldDescription(
```

Comment:
shouldnt this be private? it was on master

Reply:
fixed. My bad did not catch that one. thank you